Sources/TuistGraph/Graph/GraphDependency.swift

```diff
@@ -97,6 +97,19 @@ public enum GraphDependency: Hashable, CustomStringConvertible, Comparable, Coda
         }
     }
 
+    public var isNotStaticLibrary: Bool {
+        switch self {
+        case let .xcframework(path: _, infoPlist: _, primaryBinaryPath: binaryPath, linking: _):
+            return binaryPath.extension != "a"
```

Comment:
`linking` offers a stronger type to distinguish between dynamic frameworks and static libraries e.g.

```swift
switch linking {
  case .static:
      return false
  case .dynamic:
      return true
}
```

Reply:
I only care about static libraries though. Static frameworks are fine!